### ISSUE TITLE
Example '.cflintrc` actually doesn't run anything. 

### ISSUE THREAD

**Issue by cybersonic** [cflint/CFLint#560]
The example `.cflintrc` will not return you any errors even if you do have errors in your cfml as it has a structure in the includes key. 

This example (cleaned so no more default errors are shown): 
```
{
    "rule" : [ ],
    "excludes" : [ ],
    "includes" : [ {
        
    } ],
    "inheritParent" : false,
    
    "parameters" : {}
}
```
Has an empty structure in the includes. So without knowing the full documentation, this is saying "Do not include any rule" it seems. So if I run this on a very error prone CFML page I get no problems (silently).

Removing the `{}` it now runs all the rules. 

So this either means that we need to have some error when there is an empty structure such as:

> *  No rules to run have been defined 

Or it needs to be removed from the example in the readme?

**Comment by ryaneberly** [comment]
Does it make sense to support something like 

      npm init
      eslint --init

from the command line (and the api for IDE's).  

This could create a shell .cflintrc file in the current directory.  It could include a few simple questions to get the basic configuration the user wants.  Rule groups could be used.

I'm thinking something like

       java -jar cflint-n.n.n-all.jar -init

and perhaps
 
      java -jar cflint-n.n.n-all.jar -ignore VAR_INVALID_NAME

the 2nd would be an update

Thoughts?

**Comment by cybersonic** [comment]
Yeah definitely. If this can be mixed with commandbox so that all you do is:

box install commandbox-cflint 
box cflint init 


Mark Drew

**Comment by ryaneberly** [comment]
That's clean.   Does commandbox already implement

box cflint <args>

?
I haven't used it.

**Comment by cybersonic** [comment]
Yeah, I haven’t used this version yet but will be doing so shortly


https://www.forgebox.io/view/commandbox-cflint


Mark Drew

**Comment by KamasamaK** [comment]
An `init` would be very useful. The [NPM wrapper](https://www.npmjs.com/package/cflint) has this already. I also implemented a couple commands in my VS Code extension that do that.
